securedrop/tests/functional/functional_test.py

```diff
@@ -33,20 +34,69 @@ class FunctionalTest():
 
     def _unused_port(self):
         s = socket.socket()
-        s.bind(("localhost", 0))
+        s.bind(('127.0.0.1', 0))
         port = s.getsockname()[1]
         s.close()
         return port
 
-    def _create_webdriver(self):
-        log_file = open(join(LOG_DIR, 'firefox.log'), 'a')
-        log_file.write(
-            '\n\n[%s] Running Functional Tests\n' % str(
-                datetime.now()))
-        log_file.flush()
+
+    def _create_tor_browser_webdriver(self, abs_log_file_path):
+        # Creating the TorBrowser driver changes the working directory,
+        # so we need to save and then restore the current one.
+        old_dir = os.getcwd()
+
+        # Read the TorBrowser location from Ansible
```

Comment:
Why not just hardcode the static directory TBB lives under and get rid of this suite as well as the task that writes this file above?